stages/org.osbuild.dnf5.sbom.spdx.meta.json

```diff
@@ -0,0 +1,59 @@
+{
+  "summary": "Generate SPDX SBOM document for the installed packages.",
```

Comment:
I'd change the summaries for this and the dnf4 version to include 'DNF4' and 'DNF5' in them so it's extra totally clear which one it uses. It's kinda buried in the description. It's included in the stage name of course, but if someone is just reading the summary by it self it is the same as the DNF4 version.

Reply:
I'll submit a follow-up since the PR is already approved and I don't want to risk hunting for approvals in the pre-holiday week :innocent: